lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -310,7 +310,15 @@ def add_authentication(doc)
       def add_auth_purchase_params(doc, money, payment_method, options)
         doc.orderId(truncate(options[:order_id], 24))
         doc.amount(money)
-        add_order_source(doc, payment_method, options)
+
+        if options.dig(:stored_credential, :initial_transaction) == false
+          # orderSource needs to be added at the top of doc and
+          # processingType near the end
+          source_for_subsequent_stored_credential_txns(doc, options)
+        else
+          add_order_source(doc, payment_method, options)
```

Comment:
order_source is a GSF, would this change make it so that a merchant would no longer be able to override our stored creds framework with the GSF for non-initial transactions now?

Reply:
Yes, that's correct and this has also been approved by Litle.

Reply:
Is it possible to preserve `order_source ` being preferred? I'm worried this might be a breaking change since we are now preventing the merchant from overriding if they engage in our stored creds flow and it's the initial_transaction

Reply:
that makes sense, I'll update this part.

Reply:
Actually looking at this code again we don't allow customer to override order_source currently for stored_credentials, https://github.com/activemerchant/active_merchant/pull/4903/files#diff-05db1124b764e8e57c13d2fca5b757cfde7c92a9839255e4a829ef1f90f3d7fcL497

Reply:
oh well that's good!